FutureFeatures.md

```diff
@@ -273,14 +273,19 @@ quadruple precision.
 
 ## Floating point library intrinsics
 
-These operations aren't needed because they can be implemented in WebAssembly
-code and linked into WebAssembly modules as at small size cost, and this avoids
-a non-trivial specification burden of their semantics/precision. Adding these
-intrinsics would allow for better high-level backend optimization of these
-intrinsics that require builtin knowledge of their semantics. On the other
-hand, a code generator may continue to statically link in its own
-implementation since this provides greater control over precision/performance
-tradeoffs.
+These operations aren't needed for the MVP because they can be implemented in
+WebAssembly code and linked into WebAssembly modules at small size cost (as is
+traditionally done through C libraries such as libm). This avoids a non-trivial
+specification burden for their semantics, precision, and performance.
+
+[Dynamic linking](FutureFeatures.md#dynamic-linking) will allow caching of libm,
+making this sharing cost trivial.
```

Comment:
Are you suggesting that I change the top of the paragraph "at small size cost" or the bottom "making this sharing cost trivial"? Agreed it's very small and will become even smaller! I'm keeping these mentions here because they were part of the discussions we had originally, where folks were worried of the size overhead.